src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -460,23 +461,17 @@ def cancel_jobs(self, job_id_list: List[str]) -> dict:
         error_states = dict()
         for job_id in job_ids:
             if not self.get_job(job_id).was_terminal():
-                cancel_err = self._cancel_job(job_id)
-                if cancel_err:
-                    error_states[job_id] = cancel_err
+                error = self._cancel_job(job_id)
+                if error:
+                    error_states[job_id] = error.message
 
         job_states = self._construct_job_output_state_set(job_ids)
 
         for job_id in error_ids:
             job_states[job_id] = get_error_output_state(job_id)
 
         for job_id in error_states:
-            error = error_states[job_id]
-            job_states[job_id]["error"] = {
-                "name": error.name,
-                "message": error.message,
-                "error": error.error,
-                "code": error.code,
-            }
+            job_states[job_id]["error"] = error_states[job_id]
```

Comment:
return an error string rather than an object -- in line with how other message types return errors